libr/util/file.c

```diff
@@ -1387,7 +1387,7 @@ R_API bool r_file_copy(const char *src, const char *dst) {
 #else
 	char *src2 = r_str_replace (strdup (src), "'", "\\'", 1);
 	char *dst2 = r_str_replace (strdup (dst), "'", "\\'", 1);
-	int rc = r_sys_cmdf ("cp -f '%s' '%s'", src2, dst2);
+	int rc = r_sys_cmdf ("cp -f %s %s", src2, dst2);
```

Comment:
this is introducing a command injection vulnerability. so better not use r_sys_cmdf here imho. so if we have to replace this better do a pure native implementation without depending on `cp`

Reply:
Yes, quite right, user input sanitation and all